api-swagger-patched.yaml

```diff
@@ -324,6 +387,36 @@ parameters:
     x-example: 30
     x-ms-parameter-location: 'method'
 
+  searchParam:
+    in: query
+    name: search
+    required: false
+    type: string
+    description: Filter the results searching by this string.
+    x-example: "my-repository-name"
+    x-ms-parameter-location: 'method'
+
+  remoteOrganizationNameParam: 
+    in: path
+    name: remoteOrganizationName
+    description: 'Remote organization name'
+    required: true
+    type: string
+    x-example: "codacy"
+    x-ms-parameter-location: 'method'
+
+  providerParam:
+    in: path
+    name: provider
+    description: "Provider"
+    required: true
+    type: string
+    enum: *providerEnum
+    x-ms-enum:
```

Comment:
Not sure if this will work here. But should do not harm. If you can check the generate output.

Reply:
I did the generate. Did not warn any error 👍